smart_pet_buddy/lib/controlpanel.dart

```diff
@@ -244,15 +247,19 @@ class ControlpanelState extends State<Controlpanel> {
             Flexible(
               flex: 1,
               child: Container(
-                child: TextButton(
-                  onPressed: _forward,
-                  child: Text("forward"),
-                  key: Key('forwards'),
-                  style: ButtonStyle(
-                      backgroundColor: isForward
-                          ? MaterialStateProperty.all(Colors.red)
-                          : MaterialStateProperty.all(Colors.blue),
-                      foregroundColor: MaterialStateProperty.all(Colors.white)),
+                height: 120,
+                child: Transform.rotate(
+                  angle: 270 * pi / 180,
+                  child: IconButton(
```

Comment:
I think IconButton is a good choice here since it is not deprecated. We should avoid using FlatButton, OutlinedButton and RaisedButton overall.

Reply:
yeah, I agree 😄